README.md

```diff
@@ -72,12 +72,9 @@ ng generate component my-new-component
 ng g component my-new-component # using the alias
 
 # components support relative path generation
-# if in the directory src/app/feature/ and you run
```

Comment:
It doesn't work though? If I `cd` into a subdirectory and generate a component it doesn't actually generate the component in it.